docs/en/sensor/inertiallabs.md

```diff
@@ -0,0 +1,99 @@
+# Inertial Labs
+
+Inertial Labs designs and develops [IMU](https://inertiallabs.com/products/imu-inertial-measurement-units/), [AHRS](https://inertiallabs.com/products/ahrs/), [GNSS/INS](https://inertiallabs.com/products/ins-inertial-navigation-systems/) and [other](https://inertiallabs.com/) solutions.
+
+![INS-U](../../assets/hardware/sensors/inertial/ilabs-ins-u.png)
+
+Sensors connection is near plug-and-play, and a universal protocol is used for [all Inertial Labs sensors](https://inertiallabs.com/).
```

Comment:
"near" plug and play. Plug and play would mean that you could just attach to some labeled port and it would work without any configuration.

Reply:
Yes, I'll move "a universal protocol is used for [all Inertial Labs sensors](https://inertiallabs.com/)." up

```suggestion
```